backends/ent/annotations.go

```diff
@@ -203,7 +204,7 @@ func (backend *Backend) GetDocumentsByAnnotation(name string, values ...string)
 		return nil, errUninitializedClient
 	}
 
-	predicates := []predicate.Annotation{annotation.NameEQ(name)}
+	predicates := []predicate.Annotation{annotation.NameEQ(name), annotation.Not(annotation.HasNode())}
```

Comment:
Style nit: consistency with above change

```suggestion
	predicates := []predicate.Annotation{
		annotation.NameEQ(name), 
		annotation.Not(annotation.HasNode()),
	}
```